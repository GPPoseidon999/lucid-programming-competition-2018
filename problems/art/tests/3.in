nonsense :p
28
4
a
   |          
   |.===.     
   {}o o{}    
ooO--(_)--Ooo-
b
    ( ( (     
  '. ___ .'   
 '  (> <) '   
ooO--(_)--Ooo-
c
              
    ,,,,,     
   /(o o)\    
ooO--(_)--Ooo-
d
        ...      
   o,*,(o o)     
  8(o o)(_)Ooo   
ooO-(_)---Ooo----
e
     |"|      
    _|_|_     
    (o o)     
ooO--(_)--Ooo-
f
      _       
    _|_|_     
    (o o)     
ooO--(_)--Ooo-
g
              
   __MMM__    
    (o o)     
ooO--(_)--Ooo-
h
     ___      
    .|||.     
    (o o)     
ooO--(_)--Ooo-
i
   .      .   
 .  .:::.     
   :(o o):  . 
ooO--(_)--Ooo-
j
              
     vvv      
    (0~0)     
ooO--(_)--Ooo-
k
     ___      
    /\#/\     
   /(o o)\    
ooO--(_)--Ooo-
l
   _     _    
 o' \,=./ `o  
    (o o)     
ooO--(_)--Ooo-
m
      ___     
     /_\ `*   
    (o o)     
ooO--(_)--Ooo-
n
    _   _     
   '\\-//`    
    (o o)     
ooO--(_)--Ooo-
o
 #   ___      
 #  <_*_>     
 #  (o o)     
-8---(_)--Ooo-
p
#                 #
#=ooO=========Ooo=#
#  \\  (o o)  //  #
--------(_)--------
q
     ___      
    '/_\      
    (o o)     
ooO--(_)--Ooo-
r
     !!!      
  `  _ _  '   
 -  (OXO)  -  
ooO--(_)--Ooo-
s
              
  `  _ ,  '   
 -  (o)o)  -  
-ooO'(_)--Ooo-
t
              
    ()_()     
    (o o)     
ooO--`o'--Ooo-
u
      o       
   ` /_\ '    
  - (o o) -   
ooO--(_)--Ooo-
v
     ___      
   .'_#_`.    
   |[o o]|    
ooO--(_)--Ooo-
w
 #  #  #  #   
 #  #..#  #   
 #  #O #) #   
o#O-#(_#--#oo-
x
    _   _     
   (_)-(_)    
    (o o)     
ooO--(_)--Ooo-
y
        .     
    ,-_-|     
   ([o o])    
ooO--(_)--Ooo-
z
               
    ((__))     
     (00)      
nn--(o__o)--nn-
 
   


    
:
           
     "     
 `  ___  ' 
-ooO---Ooo-
