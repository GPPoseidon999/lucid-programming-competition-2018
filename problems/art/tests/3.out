    _   _      #   ___          _   _                        |"|          _   _                        |"|                    #                 #
   '\\-//`     #  <_*_>        '\\-//`      `  _ ,  '       _|_|_        '\\-//`      `  _ ,  '       _|_|_          "     #=ooO=========Ooo=#
    (o o)      #  (o o)         (o o)      -  (o)o)  -      (o o)         (o o)      -  (o)o)  -      (o o)      `  ___  ' #  \\  (o o)  //  #
ooO--(_)--Ooo--8---(_)--Ooo-ooO--(_)--Ooo--ooO'(_)--Ooo-ooO--(_)--Ooo-ooO--(_)--Ooo--ooO'(_)--Ooo-ooO--(_)--Ooo-    -ooO---Ooo---------(_)--------
